### PR TITLE
Remove commons.host,add Valine.js and so on.

### PR DESCRIPTION
**Pull request by Creeper2077**
1. Remove commons.host;
2. Add Valine(Commenting Platforms);
3. Add a warning about closing registration for heliohost.org
4. Update some outdated data(Arvan Cloud,Alwaysdata ,Astra).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -940,11 +940,10 @@ Table of Contents
   * [Fenix Web Server](https://preview.fenixwebserver.com) - A developer desktop app for hosting sites locally and sharing them publically (in realtime). Work however you like, using its beautiful user interface, API, and/or CLI.
   * [Free Hosting](http://freehostingnoads.net/) — Free Hosting With PHP 5, Perl, CGI, MySQL, FTP, File Manager, POP E-Mail, free sub-domains, free domain hosting, DNS Zone Editor, Web Site Statistics, FREE Online Support and many more features not offered by other free hosts.
   * [Freehostia](https://www.freehostia.com) — FreeHostia offers free hosting services incl. an industry-best Control Panel & a 1-click installation of 50+ free apps. Instant setup. No forced ads.
-  * [heliohost.org](https://www.heliohost.org) — Community powered free hosting for everyone.
+  * [heliohost.org](https://www.heliohost.org) — Community powered free hosting for everyone(Unable to register at this time).
```
Comment (ripienaar) [inline README.md:943]: Lets just remove them then

File: README.md [hunk 2]
```diff
@@ -1404,6 +1403,7 @@ Table of Contents
   * [GraphComment](https://graphcomment.com/) - GraphComment is a comments platform that helps you build an active community from website’s audience.
   * [Utterances](https://utteranc.es/) - A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!
   * [Disqus](https://disqus.com/) - Disqus is a networked community platform used by hundreds of thousands of sites all over the web.
+  * [Valine](https://valine.js.org/) wis a fast, simple & efficient Leancloud based no back end comment system.
```
Comment (ripienaar) [inline README.md:1406]: this is effectively self hosted software, so I dont think it fits the purpose of this list

File: README.md (hunk 1 above)
Comment (Creeper2077) [reply README.md]: I removed it

**Comment by ripienaar** [comment]
Sorry, I merged then reverted, Valine is self hosted software.  The *service* thats free there is Leancloud.  We cannot list every thing that is built on other free things.

**Pull request merged**